### DISCUSSION TITLE
Love from the former w3af project lead

### DISCUSSION THREAD

**Discussion by andresriancho** [security-companion/w3af#303] [General]
Thanks for using w3af's code to create w4af. I'm happy that you guys are continuing from where I left ❤️ Feel free to reach out to me via Linkedin https://www.linkedin.com/in/andres-riancho/ , we can have a couple of short online meets for Q&A if you need those to speed up your project.

**Comment by andresriancho** [comment]
cc: @security-companion @codders

**Reply by codders** [reply]
Hey! Great to see you around here :) I'll for sure be in touch

**Reply by Luongvgg** [reply]
I'm also thinking about building something from w3af but I can't install it with popular systems today like Ubuntu 20.04 or kali. Where can I find help? Surely there is a community of some kind?
